ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule/Shrinking.hs

```diff
@@ -84,15 +84,25 @@ shrinkAdversarialPeers shrink Peers {honestPeers, adversarialPeers} =
 -- We check that this operation does not changes the final state of the honest peer.
 -- NOTE: This operation makes the honest peer to end its schedule sooner, which *may*
 -- trigger disconnections when the timeout for MsgAwaitReply is reached. In those cases,
--- it is probably more pertinent to disable this timeout in tests than disabling shrinking.
+-- it is probably more pertinent to disable this timeout in tests than to disable shrinking.
```

Comment:
Probably this documentation should say if ticks are removed from the end, and if so explain why the schedule isn't invalidated. After all, the honest peer may then not send headers or blocks that it advertised to have.